crates/cargo-wdk/src/actions/package/tests.rs

```diff
@@ -2087,31 +2468,26 @@ trait TestSetupPackageExpectations {
 }
 
 impl TestPackageAction {
-    fn new(cwd: PathBuf, profile: Profile, target_arch: TargetArch, sample_class: bool) -> Self {
+    fn new(
+        cwd: PathBuf,
```

Comment:
Nit: change to `&Path`. Although this is not super important as this is only test code.